src/app/blueprint.css

```diff
@@ -1,50 +1,57 @@
-/* Styles for root page */
-.page {
-  font-family: system-ui, "Segoe UI", Roboto, Helvetica, Arial, sans-serif,
-    "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+body {
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen,
+    Ubuntu, Cantarell, 'Helvetica Neue', sans-serif;
 }
 
-/* Styles for feature component */
-
 .feature {
-  background-color: #ffffff;
+  display: flex;
+	justify-content: center;
+  max-width: calc(100% - 2rem);
+  margin: 3rem auto;
+  padding: 3rem;
+	background-color: white;
   border: 1px solid #dbdde2;
   border-radius: 8px;
-  padding: 3rem;
 
-  width: 300px;
-
-  display: grid;
-  place-content: center;
+	@media screen and (min-width: 1280px) {
+		max-width: 1024px;
+	}
 }
 
-.feature span {
-  font-size: 1rem;
-  font-weight: 600;
-}
+.grid {
+  display: grid;
+	gap: 1rem;
+  max-width: calc(100% - 2rem);
+  margin: 3rem auto;
 
+	@media screen and (min-width: 576px) {
+		grid-template-columns: repeat(2, minmax(200px, 1fr));
+	}
 
-/* Styles for grid component */
+	@media screen and (min-width: 768px) {
+		grid-template-columns: repeat(3, minmax(200px, 1fr));
+	}
 
-.grid {
-  display: flex;
-  flex-direction: row;
-  justify-content: center;
-  padding: 4rem;
-  gap: 1rem;
+	@media screen and (min-width: 1280px) {
+		max-width: 1024px;
+	}
 }
 
-
-/* Styles for teaser component */
+.grid > .feature {
+  margin: 0;
+	max-width: 100%;
+}
 
 .teaser {
+	display: flex;
+	justify-content: center;
+	padding: 7rem 1rem;
   background-color: #dbd0ff;
-  padding: 7rem;
-  display: grid;
-  place-content: center;
 }
 
-.teaser h1 {
-  text-align: center;
-  max-width: 50ch;
+.feature + .feature,
```

Comment:
Nice use of sibling selectors. I wonder if we can make it generic and just use an "owl" selector - https://css-tricks.com/lobotomized-owls/